### ISSUE TITLE
Forbidden

### ISSUE THREAD

**Issue by antaljani** [pythontester192/Canva-Pro-Lifetime-Free-Using-Python#2]
Script is working properly, howeven, when I try to open the given link, Canva ask me to login and then just saying : "It is forbidden, ask for support".
So, the link cannot usable.

**Comment by pythontester192** [comment]
Yes this might happens, just try again later, the link will be updated automatically.

**Comment by MrShadowDev** [comment]
You need to paste the link in the browser where you have your canva account. After pasting it click enter then you will have your pro canva account!

**Comment by 6Threat** [comment]
![image](https://user-images.githubusercontent.com/104936729/213308054-e2fd9068-4f74-4e10-9e67-ff5a4d26d4a5.png)
i got issue with this How i 
able to fixed this?

**Comment by 6Threat** [comment]
![image](https://user-images.githubusercontent.com/104936729/213309042-bae0d8d0-f35e-45c6-87b8-9df0d1ee28f3.png)
i always with this error. pyppeteer how was it?

**Comment by MrShadowDev** [comment]
> ![image](https://user-images.githubusercontent.com/104936729/213308054-e2fd9068-4f74-4e10-9e67-ff5a4d26d4a5.png) i got issue with this How i able to fixed this?

Just ignore it

**Comment by MrShadowDev** [comment]
> ![image](https://user-images.githubusercontent.com/104936729/213309042-bae0d8d0-f35e-45c6-87b8-9df0d1ee28f3.png) i always with this error. pyppeteer how was it?

Keep it like it is. If you are getting the canva pro link just ignore the other issues.

**Comment by MrShadowDev** [comment]
My version:

https://github.com/MrShadowDev/Free-Canva-Pro-Lifetime
